packages/server/graphql/mutations/removeAzureDevOpsAuth.ts

```diff
@@ -0,0 +1,43 @@
+import {GraphQLID, GraphQLNonNull} from 'graphql'
+import removeGitHubAuthDB from '../../postgres/queries/removeGitHubAuth'
+import {getUserId, isTeamMember} from '../../utils/authorization'
+import publish from '../../utils/publish'
+import RemoveAzureDevOpsAuthPayload from '../types/RemoveAzureDevOpsAuthPayload'
+import {SubscriptionChannel} from 'parabol-client/types/constEnums'
+import {GQLContext} from '../graphql'
+import standardError from '../../utils/standardError'
+
+const removeAzureDevOpsAuth = {
```

Comment:
-2 this should go into the `removeTeamMemberIntegrationAuth` table

Reply:
tagging @wdfox

Reply:
@wdfox made this change